admin-front/src/components/Dashboard/Dashboard.tsx

```diff
@@ -161,7 +178,7 @@ export const Dashboard = () => {
             {
               topBuyers.map((user: any, index: number) =>
                 <ListItem >
-                  <a className={classes.link} href={`https://kanvas.tzconnect.berlin/profile/${user?.address ?? 'tz1KhMoukVbwDXRZ7EUuDm7K9K5EmJSGewxd'}`} target="_blank">
+                  <a className={classes.link} href={`https://kanvas.tzconnect.berlin/profile/${user?.userAddress ?? 'tz1KhMoukVbwDXRZ7EUuDm7K9K5EmJSGewxd'}`} target="_blank">
```

Comment:
Shouldn't default to this `tz1KhMoukVbwDXRZ7EUuDm7K9K5EmJSGewxd`. If `userAddress` is null, then simply don't link?